.gitignore

```diff
@@ -22,3 +22,10 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+# Design Files
+design/*
+
+# Vite 
```

Comment:
I've added this myself, `.vite` folder is Vite Cache, usually shouldn't be committed to the repository.